services/QuillLMS/spec/controllers/skills_reports_controller_spec.rb

```diff
@@ -98,6 +98,18 @@
           expect(response).to have_http_status(:ok)
         end
       end
+
+      context 'caching' do
+        before { Rails.cache.clear }
+
+        it 'expect the first call to load the service, and the second to get data from cache' do
+          expect(teacher).to receive(:classroom_cache).twice.and_call_original
+          expect(SkillsReport::ClassroomScoresByUser).to receive(:run)
```

Comment:
Should there be a `.once` here? to make the test explicit, e.g. `receive(:run).once`